xc7/utils/prjxray_arch_import.py

```diff
@@ -311,7 +311,7 @@ def main():
             continue
         elif gridinfo.tile_type in tile_types:
             # We want to import this tile type.
-            vpr_tile_type = 'BLK_TI-{}'.format(gridinfo.tile_type)
+            vpr_tile_type = '{}'.format(gridinfo.tile_type)
```

Comment:
Why not just `vpr_tile_type = gridinfo.tile_type`?

Reply:
Done.